packages/embeds/js/src/features/blocks/inputs/number/components/NumberInput.tsx

```diff
@@ -14,24 +18,46 @@ type NumberInputProps = {
 };
 
 export const NumberInput = (props: NumberInputProps) => {
-  const [inputValue, setInputValue] = createSignal<string | number>(
-    props.defaultValue ?? "",
-  );
-  const [staticValue, bindValue, targetValue] = numberInputHelper(() =>
-    inputValue(),
-  );
+  const minValue =
+    props.block.options?.min !== undefined
+      ? Number(props.block.options.min)
+      : undefined;
+  const maxValue =
+    props.block.options?.max !== undefined
+      ? Number(props.block.options.max)
+      : undefined;
+  const stepValue =
+    props.block.options?.step !== undefined
+      ? Number(props.block.options.step)
+      : undefined;
```

Comment:
We tend to only declare variables if it's re-used at least twice. Otherwise, we can inline all of them